### PR TITLE
Microsoft.AAD: Add OuContainer Swagger

### PR DESCRIPTION
**Pull request by jihochang**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by raych1 — COMMENTED** [review]
Can you fix the lintDiff check failures? And what's the reason to add these new apis to old api version(2017-06-01)?

File: specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/oucontainer.json [hunk 1]
```diff
@@ -0,0 +1,551 @@
...
+          },
+          {
+            "$ref": "#/parameters/OuContainerNameParameter"
+          },
+          {
+            "name": "ouContainer",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/OuContainerInput"
```
Comment (ravbhatnagar) [inline specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/oucontainer.json:167]: request body and response body must be same

File: specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/oucontainer.json [hunk 2]
```diff
@@ -0,0 +1,551 @@
...
+            },
+            "description": "Properties supplied to the Update OuContainer operation."
+          }
+        ],
+        "responses": {
```
Comment (ravbhatnagar) [inline specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/oucontainer.json:172]: add "default" response to all your API response sections. please check other RP swaggers to get a sample. like Batch RP.

**Review by ravbhatnagar — COMMENTED** [review]
@jihochang few comments

**Comment by raych1** [comment]
@jianyexi , can you guide on how to fix the lintDiff error?

**Comment by jihochang** [comment]
> @jianyexi , can you guide on how to fix the lintDiff error?

I have fixed all linting issues!

File: specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/oucontainer.json (hunk 2 above)
Comment (jihochang) [reply specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/oucontainer.json:190]: this has been added.

File: specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/oucontainer.json (hunk 1 above)
Comment (jihochang) [reply specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/oucontainer.json:185]: > @jianyexi , can you guide on how to fix the lintDiff error?

I have fixed the issue.

**Comment by jihochang** [comment]
> @jianyexi , can you guide on how to fix the lintDiff error?

fixed

**Review by ravbhatnagar — COMMENTED** [review]
@jihochang one comment

**Review by raych1 — APPROVED** [review]

**Pull request merged**